MOBILE_CLIENT/COMMON/_CRITICAL/DEPENDENCY_CONFUSION/recommendation.md

```diff
@@ -1,8 +1,36 @@
-NPM packages offer the possibility to set scopes to packages. Scoped packages on the public npm registry may only be published by the user or organization associated with it, and packages within that scope may be made private. Also, scope names can be linked to a given registry.
+### 1. Create a Scoped Package:
```

Comment:
![image](https://github.com/Ostorlab/KB/assets/146742880/90c4ec6b-38e2-4737-97a7-23b22abcee41)
when you use 3 ### -> it appear very big , and in my case I did change the first to be #### and it's better

Reply:
+1. #### is better